lib/kube/grpc/grpc.go

```diff
@@ -159,7 +159,7 @@ func (s *Server) ListKubernetesResources(ctx context.Context, req *proto.ListKub
 	if req.UseSearchAsRoles || req.UsePreviewAsRoles {
 		var extraRoles []string
 		if req.UseSearchAsRoles {
-			extraRoles = append(extraRoles, userContext.Checker.GetAllowedSearchAsRoles(services.WithKubernetesRequestModeFilter(req.ResourceType))...)
```

Comment:
i removed this, i was thinking we shouldn't deny a user to at least `query` something since i think some of the kubernetes resources require you to define a namespace right?